AssetEditor/Services/VersionChecker.cs

```diff
@@ -18,8 +18,10 @@ public static void CheckVersion()
 
             try
             {
-                System.Reflection.Assembly assembly = System.Reflection.Assembly.GetExecutingAssembly();
-                FileVersionInfo fvi = FileVersionInfo.GetVersionInfo(assembly.Location);
+                //System.Reflection.Assembly assembly = System.Reflection.Assembly.GetExecutingAssembly();
```

Comment:
why was this changed? What's wrong with the old way of retrieving file version info on the .exe?